sphinx/writers/latex.py

```diff
@@ -1754,16 +1757,8 @@ def depart_image(self, node):
 
     def visit_figure(self, node):
         # type: (nodes.Node) -> None
-        ids = ''  # type: unicode
-        for id in self.pop_hyperlink_ids('figure'):
-            ids += self.hypertarget(id, anchor=False)
-        if node['ids']:
-            ids += self.hypertarget(node['ids'][0], anchor=False)
+        targets = self.hypertarget_to(node)
```

Comment:
in depart_table you used `labels`. I would propose to use `labels` here too. It is easier for me because I know this is what the LaTeX code will look like: with  `\label` and possibly a `\phantomsection` before...

Reply:
Okay :-)